.github/NEW_IA_TEMPLATE.md

```diff
@@ -0,0 +1,49 @@
+# New {IA Name} Fathead Instant Answer
+Help DuckDuckGo become the best Search Engine for {Language} developers!
+
+
+### What do I need to know?
+You'll need to know how to code in **Perl**, **Python**, **Ruby**, or **JavaScript**.
+![fathead languages](https://cloud.githubusercontent.com/assets/873785/19787916/57b4c31a-9c73-11e6-9bd9-f85c8893ec93.jpg)
+
+
+### What am I doing?
+You will write a script that scrapes or downloads the data source below, and generates an **output.txt** file containing the parsed documentation.
+
+You can learn more about Fathead, and the `output.txt` syntax [**here**](https://docs.duckduckhack.com/resources/fathead-overview.html)!
+
+
+### The data source :rocket:
+{URL to Data Source}
+
+
+### What is the Goal?
+As part of our Programming Mission, we're aiming to reach 100% Instant Answer (IA) coverage for searches related to programmiing languages by creating new Instant Answers, and improving existing ones.
+
+Here are some Fathead examples:
+
+- Ruby Docs
+    - [Code](https://github.com/duckduckgo/zeroclickinfo-fathead/tree/master/lib/fathead/ruby) | [Example Query](https://duckduckgo.com/?q=array+bsearch&ia=about) | [IA Page](https://duck.co/ia/view/ruby)
+- MDN CSS
+    - [Code](https://github.com/duckduckgo/zeroclickinfo-fathead/tree/master/lib/fathead/mdn_css) | [Example Query](https://duckduckgo.com/?q=css+background-position&ia=about) | [IA Page](https://duck.co/ia/view/mdn_css)
+
+  ![css background-position search](https://cloud.githubusercontent.com/assets/873785/19787980/cdf51566-9c73-11e6-9ef2-ac44dede62c7.png)
+
+
+[See more related Instant Answers](https://duck.co/ia?repo=fathead)
+
+
+## Get Started
+- [ ] 1) Claim this issue (Comment Below)
+- [ ] 2) Review our [Contributing Guide](https://github.com/duckduckgo/zeroclickinfo-fathead/blob/master/CONTRIBUTING.md)
+- [ ] 3) [Set up your development environment](https://docs.duckduckhack.com/welcome/setup-dev-environment.html), then fork this repository
+- [ ] 4) Create the Fathead
+- [ ] 5) Create a Pull Request
+- [ ] 6) Ping @{Language Leader Handle} for a review
+
+
+## Questions?
+- Join [DuckDuckHack Slack](https://quackslack.herokuapp.com/) and ask questions
+- Ping one of the [Community Leaders](https://duck.co/help/community/community-leaders)
```

Comment:
Not sure about including this. It would be nice to keep all questions in the open if possible, so others can benefit from the discussion and answers.